data_import/models.py

```diff
@@ -81,29 +69,11 @@ class DataFileManager(models.Manager):
     pre_delete signal connected correctly.
     """
 
-    def for_user(self, user):
```

Comment:
Moved to ProjectDataFile as a more natural place for it, since this manager was filtering on a field of that model.